packages/embark-i18n/src/index.ts

```diff
@@ -20,7 +20,7 @@ enum SupportedLanguage {
 const DEFAULT_LANGUAGE = SupportedLanguage.En;
 
 i18n.configure({
-  directory: path.join(__dirname, "../../../../", "locales"),
+  directory: path.join(__dirname, "../", "locales"),
   locales: Object.values(SupportedLanguage),
   register: global,
```

Comment:
Can we consider dropping global registration as part of the refactor?

The idea is that `embark-i18n` would export the `__` of an `i18n` object that's already been configured with our locales. Then, every module that needs to use `__` would explicitly require/import it from `embark-i18n` instead of relying on a global `__`.

Reply:
@michaelsbradleyjr I've looked into this as per https://www.npmjs.com/package/i18n#some-words-on-register-option

Exposed `__` via `embark-i18n` and deregistered `__` from the global scope. Unfortunately I can't get pass TypeScript due to several typecheck issues.

As this PR here is a blocker for me I'm putting this separate effort aside so I can move on with the blockchain-process module.

However, I've pushed my work here: https://github.com/embark-framework/embark/tree/refactor/i18n-embark-i18n

Basically just needs to pass typecheck and some additional manual tests.

Reply:
Took your work forward and turned it into a PR. See: #1597.